common/lib/realm_common.dart

```diff
@@ -16,8 +16,7 @@
 //
 ////////////////////////////////////////////////////////////////////////////////
 
-import 'package:realm_generator/src/tmp/test.dart';
-
-var c = new Car();
-var make = c.make;
+library realm_common;
```

Comment:
I missed this one. We should not declare a library name since this is what they suggest.

> When the library directive isn’t specified, a unique tag is generated for each library based on its path and filename. Therefore, we suggest that you omit the library directive from your code unless you plan to generate library-level documentation.

Reply:
Fixed